### ISSUE TITLE
when change materail to CompositeMaterial then Plane disappear

### ISSUE THREAD

**Issue by pdelia** [pdelia/away3d#46]
#10 Issue by __GoogleCodeExporter__, created on: 2015-04-24T07:51:03Z

```
What steps will reproduce the problem?
1. create CompositeMaterial
2. create Plane with another material
3. change Plane material to CompositeMaterial

What is the expected output?

Plane with new material (CompositeMaterial)

What do you see instead?

nothing ) (Plane disappear)

What version of the product are you using? On what operating system?

away3d rev. 586. WinXP. Flash CS3

Please provide any additional information below.

var faceOverMaterial:CompositeMaterial = new
CompositeMaterial({materials:[faceMaterial, overMaterial]});

_model = new Plane({material:faceOverMaterial, back:backMaterial,
width:700, height:1000, segmentsW:1, segmentsH:1});

private function onMouseOver(param_event)
{
        param_event.target.material = faceOverMaterial; // Not Working!

}


private function onMouseOver(param_event)
{
        param_event.target.material = notComposeMaterial; // All ok

} 
```

Original issue reported on code.google.com by `nauro...@gmail.com` on 24 Jun 2008 at 4:30

**Comment by pdelia** [comment]
Comment by __GoogleCodeExporter__, created on: 2015-04-24T07:51:03Z



Original comment by `rob.bate...@gmail.com` on 11 Nov 2008 at 4:05
* Changed state: **Accepted**

**Comment by pdelia** [comment]
Comment by __GoogleCodeExporter__, created on: 2015-04-24T07:51:03Z

```
fixed in revision 1111, version 2.3.1
```

Original comment by `rob.bate...@gmail.com` on 17 Feb 2009 at 1:00
* Changed state: **Fixed**
